recipes/libconfig/all/conanfile.py

```diff
@@ -0,0 +1,81 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class LibconfigConan(ConanFile):
+    name = "libconfig"
+    license = "LGPL-2.1"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://hyperrealm.github.io/libconfig/"
+    description = "C/C++ library for processing configuration files"
+    topics = ("conan", "conf", "config", "cfg", "configuration")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False]}
+    default_options = {"shared": False,
+                       "fPIC": True}
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake",
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
```

Comment:
Should this be prefered over the _old_ way?

Reply:
sure, why not?

Reply:
awesome 👍

Reply:
Yes. We added `strip_root` for this propose ;)